Source/Modules/php.cxx

```diff
@@ -441,6 +457,9 @@ class PHP : public Language {
     Printf(s_header, "}\n");
     Printf(s_header, "#endif\n\n");
 
+    Printf(s_header,"#ifdef __GNUG__\n#define SWIG_remove(zv) delete zv\n");
+    Printf(s_header,"#else\n#define SWIG_remove(zv) free(zv)\n#endif\n\n");
```

Comment:
I'm guessing you want to test `#ifdef __cplusplus` here - making it conditional on a particular compiler doesn't seem right.